src/layouts/index.jsx

```diff
@@ -51,19 +49,26 @@ class Layout extends React.Component {
     }, []);
   };
 
-  getSections = () => {
+  getSections = id => {
     const { pages } = this.state;
-    const sections = [];
+    const { investigations } = this.props;
+    const investigation = find(investigations, { id });
+    const defaultSection = [{ sectionName: 'default', pages: [] }];
+    const sections =
+      get(investigation, 'sections', defaultSection) || defaultSection;
```

Comment:
updated the logic in here, `investigation` is undefined in one prod build case for the 404 page which created exceptions trying to get `sections` out of it. Lodash `get` provides the option for a default fallback and protects against undefined exceptions, the `||` operator then handles cases where `sections` is null.